src/site/xdoc/cmdline.xml.vm

```diff
@@ -450,6 +450,355 @@ java -D&lt;property&gt;=&lt;value&gt;  \
+                message="Fall through from previous branch of switch statement"
+                source="com.puppycrawl.tools.checkstyle.checks.coding.FallThroughCheck"/&gt;
+        &lt;/file&gt;
+    &lt;/checkstyle&gt;
+    </code></pre></div>
+
+      <h3>3. Specifying a Properties File (-p)</h3>
+      <p><strong>Goal:</strong> Load custom properties into Checkstyle from a properties file.</p>
+
+      <p><strong>Properties File (checkstyle.properties):</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-properties">
+      checkstyle.header.file=header.txt
+      </code></pre></div>
+
+      <p><strong>Command:</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-bash">
+      $ java -jar checkstyle-10.18.1-all.jar -c config.xml -p checkstyle.properties Test.java
+      </code></pre></div>
+
+      <p><strong>Output:</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-bash">
+      Starting audit...
+      [ERROR] Test.java:1: File does not contain required header. [Header]
+      Audit done.
+      Checkstyle ends with 1 errors.
+      </code></pre></div>
+
+      <h3>4. Specifying an Output File (-o)</h3>
+      <p><strong>Goal:</strong> Save Checkstyle results to a file.</p>
+
+      <p><strong>Command:</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-bash">
+      $ java -jar checkstyle-10.18.1-all.jar -c config.xml -o results.txt Test.java
+      </code></pre></div>
+
+      <p><strong>Output (inside results.txt):</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-bash">
+      Starting audit...
+      [ERROR] Test.java:9:9: Fall through from previous branch of switch statement [FallThrough]
+      Audit done.
+      Checkstyle ends with 1 errors.
+      </code></pre></div>
+
+      <h3>5. Printing XPath Suppressions (-s)</h3>
+      <p><strong>Goal:</strong> Generate XPath suppressions for a specific line and column.</p>
+
+      <p><strong>Command:</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-bash">
+      $ java -jar checkstyle-10.18.1-all.jar -s 9:9 Test.java
+      </code></pre></div>
+
+      <p><strong>Output:</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-xml">
+      Suppression XPath query: &lt;suppress xpath="//CASE_GROUP" files="Test.java"/&gt;
+      </code></pre></div>
+
+      <h3>6. Generating Suppressions XML (-g)</h3>
+      <p><strong>Goal:</strong> Generate an XML file containing suppressions for all violations.</p>
+
+      <p><strong>Command:</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-bash">
+      $ java -jar checkstyle-10.18.1-all.jar -c config.xml -g -o suppressions.xml Test.java
+      </code></pre></div>
+
+      <p><strong>Output (inside suppressions.xml):</strong></p>
```

Comment:
`Content of suppressions.xml`

Please do update to all other wording like this